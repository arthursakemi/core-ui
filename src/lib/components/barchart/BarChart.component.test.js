import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/barchart/__snapshots__"
});
