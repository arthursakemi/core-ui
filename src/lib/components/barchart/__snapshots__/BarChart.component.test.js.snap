// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots BarChart Default 1`] = `
<div
  className="sc-bwzfXH dNCSlp"
>
  <h3
    className="sc-htpNat iELaDV"
  >
    Vertical Stacked Bar Chart Demo 
  </h3>
  <div
    className="sc-bdVaJa bDWFJH sc-vegachart"
    id="vis_vertical_stacked"
  />
  <h3
    className="sc-htpNat iELaDV"
  >
    Horizontal Stacked Bar Chart Demo 
  </h3>
  <div
    className="sc-bdVaJa bDWFJH sc-vegachart"
    id="vis_horizontal_stacked"
  />
  <h3
    className="sc-htpNat iELaDV"
  >
    Vertical Bar Chart Demo 
  </h3>
  <div
    className="sc-bdVaJa bDWFJH sc-vegachart"
    id="vis_vertical"
  />
  <h3
    className="sc-htpNat iELaDV"
  >
    Horizontal Bar Chart Demo 
  </h3>
  <div
    className="sc-bdVaJa bDWFJH sc-vegachart"
    id="vis_horizontal"
  />
</div>
`;
